packages/ui/rating/style.ts

```diff
@@ -110,4 +110,24 @@ export const Wrapper = styled.div`
   .text-progress {
     font-size: 14px;
   }
+
+  @media only screen and (max-width: 600px) {
```

Comment:
you can utilize the constants defined in the Breakpoints enum 
@media (max-width: ${Breakpoints.md}) {}

Reply:
@TreshAnn , breakpoints enum utilized and pushed. Thanks.